src/jobs-worker/FloraJobs/Runner.hs

```diff
@@ -189,3 +196,15 @@ assignNamespace =
             then PackageAlternative (Namespace "haskell") p
             else PackageAlternative (Namespace "hackage") p
       )
+
+fetchFundingInformation :: Text -> Text -> JobsRunner ()
+fetchFundingInformation owner repo = do
+  JobsRunnerEnv{ mGithubToken } <- Reader.ask @JobsRunnerEnv
+  case mGithubToken of
+    Nothing -> pure ()
```

Comment:
In retrospect we should also perform such a check in [fetchMetadataHandler](https://github.com/flora-pm/flora-server/blob/development/src/web/FloraWeb/Pages/Server/Admin.hs#L72) so that we don't queue jobs for nothing in the absence of the GitHub Token

Reply:
I will have a look at sorting this!